web-services/handler.js

```diff
@@ -43,7 +69,50 @@ module.exports.getBoard = async (event, context) => {
     }),            
   };  
 };
-
+  
+//--------------------------------------------------
+//The function will get card data from Trello, 
+//and put them back to the /chardb endpoint 
+//--------------------------------------------------
+module.exports.getAction = async (event, context) => {
+  try{ 
+    var urlPath = 'https://api.trello.com/1/boards/5b33a8f601c94a9c9b0e71d8/actions?';
+    var params = {                  
+      limit: 1000,
+      since: '2018-10-01',
+      fields: 'date',
+      filter: 'createCard',
+      memberCreator: false,
+      key: process.env.KEY,
+      token: process.env.TOKEN
+    }
+    //get created cards info from Trello API   
+    var tb1 = await getData(urlPath, params);
+    
+    urlPath = 'https://api.trello.com/1/lists/5b33d1f83c2f9cf65b0d80dd/cards?'
+    params = {                  
+      fields: 'dateLastActivity',
+      key: process.env.KEY,
+      token: process.env.TOKEN
+    }
+    //get resolved cards info from Trello API  
+    var tb2 = await getData(urlPath, params);     
+  }catch (error){
+    console.log();
+  } 
+  //process and return data for chartdb endpoint 
+  return {
+    statusCode: 200,
```

Comment:
You might not want this to always be 200 if you had an error (i.e. a 500 would be more appropriate). There is a convention built into libraries such as axios which know a non-200/300 status code indicates an error of some kind.